.travis.yml

```diff
@@ -9,4 +9,5 @@ install: "pip install -r requirements_dev.txt"
 # command to run tests, e.g. python setup.py test
 script:
   - python setup.py test
+  - flake8 --version
```

Comment:
This is probably unnecessary, the version is printed when installing.

Reply:
I can take it out, but for me it was *very* useful. 
I was doing some other development here and flake was telling me other stuff. If you check it out I have PR called `flake8 is happy again` and another one called `Revert "flake8 is happy again"`, because different flake versions give me different output, so that's the easy way to know what sublibraries `flake8` has. 

I applied the same trick here https://github.com/jstasiak/python-zeroconf/pulls and that's why I am the only one with a passing build. I would keep it. It does not affect the end user and make the devs happier.

Reply:
Yeah, that's true, I just generally dislike putting too much stuff in PRs. This is fine though, thanks.